massa-bootstrap/src/server_binder.rs

```diff
@@ -201,28 +218,43 @@ impl<D: Duplex + 'static> BootstrapServerBinder<D> {
 
     #[allow(dead_code)]
```

Comment:
Can be removed

Reply:
https://github.com/massalabs/massa/pull/3745/commits/5a990a2ad6be4038c2c9b6a85e0fc04c404f280e